README.md

```diff
@@ -333,27 +333,49 @@ React Denmark Map exports several components, each being a map of Denmark with d
-| `onMouseEnter`        | A function that is invoked when the mouse enters an area.                                               | (area: AreaType) => void                                                      |                                                |
-| `onMouseLeave`        | A function that is invoked when the mouse leaves an area.                                               | (area: AreaType) => void                                                      |                                                |
+| `className` | The `className` applied directly to the `<svg>` element. | string | "" |
+| `style` | The style object applied directly to the `<svg>` element. | CSSProperties<sup>\*</sup> | {} |
+| `viewbox` | The viewbox applied directly to the `<svg>` element. | { top?: number, left?: number, width?: number, height?: number } | { top: 0, left: 0, width: 1000, height: 1215 } |
+| `color` | The default color of each municipality. | CSSProperties["fill"] | #ccc |
+| `clickable` | Whether the clickable styles should be applied to the `<path>` element (the area). | boolean | false |
+| `hoverable` | Whether the hoverable styles should be applied to the `<path>` element (the area). | boolean | true |
+| `showTooltip` | Whether to render the tooltip. | boolean | true |
+| `bornholmAltPosition` | Whether to render Bornholm closer to Zealand in the `Municipalities` component.\*\* | boolean | false |
+| `laesoeAltPosition` | Whether to render Læsø slightly closer to Jutland in the `Municipalities` component. | boolean | false |
+| `anholtAltPosition` | Whether to render Anholt closer to Jutland in the `Municipalities` component. | boolean | false |
+| `zoomable` | Whether you should be able to zoom on the map. | boolean | true |
+| `customZoomControls` | A functional component that returns custom zoom controls. Rendered top-right. | (props: { onZoomIn(): void; onZoomOut(): void }) => ReactNode | |
+| `customTooltip` | A functional component that returns a custom tooltip. | (props: { area: AreaType<sup>\*\*\*</sup> }) => ReactNode | |
+| `customizeAreas` | A function that is invoked for every area and returns an object to style the area. | (area: AreaType) => { className?: string, style? CSSProperties } \| undefined | |
+| `filterAreas` | A function that is invoked for every area that avoids rendering the area if the function returns false. | (area: AreaType) => boolean | |
+| `onClick` | A function that is invoked when an area is clicked. | (area: AreaType) => void | |
+| `onHover` | A function that is invoked when an area is hovered. | (area: AreaType) => void | |
+| `onMouseEnter` | A function that is invoked when the mouse enters an area. | (area: AreaType) => void | |
+| `onMouseLeave` | A function that is invoked when the mouse leaves an area. | (area: AreaType) => void | |
+
+> > > > > > > 808b49f367902cff4efc069a900a1a3fbc691aeb
```

Comment:
Looks like something has gone wrong while fixing conflicts because the content is duplicated and the descriptions are also included. Always remember to pull from the starting branch before branching out to avoid situations like these (; The easiest way to fix it is to go into your editor and replace the above text with the text that was here before from the `v2` branch ([this file](https://github.com/MartinP460/react-denmark-map/blob/v2/README.md)). That should reset it and then you can remove the three props (: Also remember to remove the branch descriptions (i.e. "> > > > > ..." and "< < < < head"). Merge conflicts can be pretty horrible but it's worth it to try and fix them in your own editor since it has better syntax highlighting and linting.

Reply:
As you can see it had the latest changes from v2. You can find changes from other merge pull requests. 

It was just a matter of bad handling on conflicts which I also didnt see the result.